packages/@aws-cdk/aws-iot/package.json

```diff
@@ -100,6 +100,11 @@
   "engines": {
     "node": ">= 14.15.0"
   },
+  "awslint": {
+    "exclude": [
+      "no-unused-type:@aws-cdk/aws-iot.ActionConfig"
```

Comment:
What's causing this violation? I generally don't think we should be adding lint exclusions but instead should make sure the code aligns to them.

Reply:
@TheRealAmazonKendra 
I've researched it.
It was caused by addind `@internal` to method `IAction._bind()`, the message is:
```
error: [awslint:no-unused-type:@aws-cdk/aws-iot.ActionConfig] type or enum is not used by exported classes (declared at lib/action.ts:20)
```

It appears that `IAction._bind()` is no longer listed in [`Type.ownMethods`](https://github.com/aws/aws-cdk/blob/598dc01118f46b42fc0fb6b57bec4bd5218a3db8/packages/awslint/lib/rules/no-unused-type.ts#L59).
This caused `ActionConfig` to be markded as unused-type.

Changing `ActionConfig` to not export will suppress this violation. However, if we do so, `ActionConfig` will not be able to be used in `aws-iot-actions`.

Like this, when an interfaces is extended across packages, adding lint exclusions seems reasonable.
WDYT?